source/ed/ed2nav.cpp

```diff
@@ -165,6 +164,42 @@ struct FindAdminWithCities {
     }
 };
 
+bool write_data_to_file(const std::string& output_filename, navitia::type::Data& data) {
+    auto logger = log4cplus::Logger::getInstance("log");
+    std::string temp_output_filename = output_filename + ".temp";
+    std::string backup_output_filename = output_filename;
```

Comment:
Shouldn't it be 
```suggestion
    std::string backup_output_filename = output_filename + ".bak";
```
?
(then used below)

Reply:
I didn't make the names very clear but I wanted to store the old output_filename in back_output_filename before adding ".bak" to the old output_filename. The new data file name with temp_output_filename is then renamed into back_output_filename, or, if you prefer, the original output_filename. Hum, sorry I'm not sure if that really is clearer..